configs/machines/generic.yaml

```diff
@@ -1,44 +1,154 @@
-        focipool: "/path/not/set/for/generic/yaml/"
-        pool: "/path/not/set/for/generic/yaml"
+#        focipool: "<set_path_to_pool_dir>"
+#        pool: "<set_path_to_pool_dir"
+#
+# Is this a dublicate ??? Can be set in runscript.
+pool_dir:  "<set_path_to_pool_dir>"
+#
+# Specify compiler (e.g. intel2022_openmpi)
+compiler_mpi: <compiler>
 
-pool_dir:  "/path/not/set/for/generic/yaml/"
+# Choose libraries. Define further libraries (depending on iolibraries value set in the next line) below in section choose_iolibraries
+iolibraries: system_libs
 
-# standard environment setup
+# Setup environment
+# Load necessary modules (e.g. git, cdo, nco). Edit and extent the following lines.
 module_actions:
-   - "load something"
+        - "purge"
+        - "load <module>"
+#        - "load <module>"
+#        - "load <module>"
+#
+# Export environment variables. Uncomment and extend the following lines to export needed environment variables.
+#export_vars:
+#        # Locale Settings
+#        LC_ALL: en_US.UTF-8
+#        <variables>: <value>
+#
+# Variables needed exclusively during runtime
+# NOTE: runtime_environment_changes does not work from the machine files
+#choose_general.run_or_compile:
+#    runtime:
+#        add_export_vars:
+#            var1: ...
+#
+# Set information depending on choosen value for variable compiler_mpi set above. Uncomment and extend the following lines.
+#choose_compiler_mpi:
+#       <compiler>:
+#           <variable>: <value>
+#           # load additional modules specific for compiler
+#           add_module_actions:
+#               - "load <module1>"
+#               - "load <module2>"
+#               - "load <module3>"
+#                 # add additional variables to exported: change and extend following lines:
+#           add_export_vars:
+#               FC: <mpif90>
+#               F77: <mpif90>
+#               MPICC: <mpicc>
+#               MPIFC: <mpif90>
+#               CC: <mpicc>
+#               CXX: <mpicxx>
+
+# Set information depending on choosen value for variable iolibraries (see above). Uncomment and extend the following lines.
+#choose_iolibraries:
+#    system_libs:
+#        # Load addition modules:
+#        add_module_actions:
+#            - "load <module>"
+#            - "load <module>"
+#            - "load <module>"
+#
+#        # add additional variables to exported: change and extend following lines:
+#        add_export_vars:
+#            <variable>: <value>
 
-# some yamls use computer.fc, etc to identify the compiler, so we need to add them
-fc: "unset"
-cc: "unset"
-mpifc: "unset"
-mpicc: "unset"
-cxx: "unset"
+#
+## some yamls use computer.fc, etc to identify the compiler, so we need to add them
+fc: "$FC"
+cc: "$CC"
+mpifc: "$MPIFC"
+mpicc: "$MPICC"
+cxx: "$CXX"
+#
+#
+launcher_flags: "-l" # --distribution=block:cyclic"
 
+# Choose another configuration file, that should be evaluated (e.g. slurm.yaml)
 further_reading:
         - batch_system/slurm.yaml
```

Comment:
We also support PBS (in Aleph for example). So this line could also be `- batch_system/pbs.yaml`

Reply:
Added pbs.yaml in the comment.